e2e/cypress/integration/enterprise/ldap_group/group_mentions_spec.js

```diff
@@ -0,0 +1,278 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @enterprise @ldap_group
+
+import * as TIMEOUTS from '../../../fixtures/timeouts';
+import users from '../../../fixtures/ldap_users.json';
+
+let groupID;
+let boardUser;
+
+// Goes to the groups page for the group specified by id as sysadmin
+const navigateToGroup = (id) => {
+    // # Login as sysadmin and visit board group page, and wait until board user is visible
+    cy.apiLogin('sysadmin');
+    cy.visit(`/admin_console/user_management/groups/${id}`);
+
+    // # Scroll users list into view and then make sure it has loaded before scrolling back to the top
+    cy.get('#group_users').scrollIntoView();
+    cy.findByText(boardUser.email).should('be.visible');
+    cy.get('#group_profile').scrollIntoView();
+};
```

Comment:
@saturninoabril I waited for the boardUser to be visible here instead of doing it in multiple places before clicking the switch